woocommerce/woocommerce-android
# Issue/5460 simple payments cardreader take2

This PR implements taking a card payment from the simple payments flow. In doing this I had three options:

1. Rewrite the card reader flow so it could be more easily shared from multiple navigation graphs
2. Duplicate the card reader flow and adapt it for simple payments
3. Have the simple payments flow show order detail and tell it to launch the card reader flow

I ruled out the first option because it's a large task and we're on a tight schedule. I originally chose the third option and implemented it [here](https://github.com/woocommerce/woocommerce-android/pull/5623), but I decided that was a terrible user experience. So in this PR I implemented the second option, despite not liking the code duplication, using a separate navigation graph for the simple payments flow.

To test:

* Go to the order list and create a simple payment
* Tap Take Payment
* Choose Card payment
* Verify the payment flow works as expected

It would also be good to verify that the card reader flows in the settings and order detail still work (they worked fine for me).

Only one reviewer is needed, but it might be good if @malinajirka or @kidinov took a quick look since they're the most familiar with the card reader codebase.
